content/posts/player_grafton.md

```diff
@@ -7,13 +7,14 @@ featuredImage: >-
   https://cdn.discordapp.com/attachments/641517745711611904/641520162994126848/DSC_0507.JPG
 excerpt: >-
   This is placeholder text that our web designers put here to make sure words
-  appear properly on your website. 
```

Comment:
Check out https://editorconfig.org/ to clean up whitespace at the end of your lines